platform/javascript/js/libs/library_godot_input.js

```diff
@@ -424,9 +424,9 @@ const GodotInput = {
 			for (let i = 0; i < touches.length; i++) {
 				const touch = touches[i];
 				const pos = GodotInput.computePosition(touch, rect);
-				GodotRuntime.setHeapValue(coords + (i * 2), pos[0], 'double');
-				GodotRuntime.setHeapValue(coords + (i * 2 + 8), pos[1], 'double');
-				GodotRuntime.setHeapValue(ids + i, touch.identifier, 'i32');
+				GodotRuntime.setHeapValue(coords + (i * 2 * 8), pos[0], 'double');
+				GodotRuntime.setHeapValue(coords + (i * 2 * 8) + 8, pos[1], 'double');
```

Comment:
For the record, this could also have been:
```suggestion
				GodotRuntime.setHeapValue(coords + (i * 2) * 8, pos[0], 'double');
				GodotRuntime.setHeapValue(coords + (i * 2 + 1) * 8, pos[1], 'double');
```
to clarify that it's "index" * "size of double".
But it's also fine as is, it's mostly a matter of taste.

Reply:
I like your style :)